packages/arb-token-bridge-ui/src/util/CommonUtils.ts

```diff
@@ -16,5 +16,8 @@ export function shortenTxHash(txHash: string) {
   )}`
 }
 
+export const isE2eTestingEnvironment =
+  typeof window !== 'undefined' && !!window.Cypress
+
 export const isTestingEnvironment =
```

Comment:
There's 2 places where we use `isTestingEnvironment`:
* in `ProjectListing.tsx` to display some dummy data
* in `setup.ts` to configure chains

I think what we really need is `isDevelopmentEnvironment` because both of these use cases really depend on that. Having both `isE2eTestingEnvironment` and `isTestingEnvironment` is confusing.

Reply:
Ah, good name `isDevelopmentEnvironment` ! Even I felt having `isE2eTestingEnvironment` and `isTestingEnvironment` was not optimal but couldn't come up with the perfect naming. Ty!